Embedded Metadata.js

```diff
@@ -571,8 +573,18 @@ function addLowQualityMetadata(doc, newItem) {
 		}
 	}
 	
-	if(!newItem.creators.length) getAuthorFromByline(doc, newItem);
-	
+	if(!newItem.creators.length) {
+		//the authors in the standard W3 author tag are safer than byline guessing
+		var w3authors = ZU.xpath(doc, '//meta[@name="author"]');
+		if (w3authors.length>0){
+			for (var i = 0; i<w3authors.length; i++){
```

Comment:
I think the `if` is not needed here, because if the length is null you still can do the (empty) for-loop.